near/near-wallet
# Migration Modal using batch import route

In this PR:

- Wallet migration modal prompt
- Wallet migration encode account data to batch import url
- Allow select different wallet types

Deployment Preview URL: https://my-near-wallet-pr-2752.onrender.com/.

### How to test:
1. Click the link above to open up the deploy preview.
2. Import your existing account or create a new one.
3. You should see a banner at the top of the page prompting you to migrate your account. Follow the instructions therein to migrate your account to mynearwallet.
